website/content/api-docs/acl/roles.mdx

```diff
@@ -0,0 +1,340 @@
+---
+layout: api
+page_title: ACL Roles - HTTP API
+description: The /acl/roles endpoints are used to configure and manage ACL roles.
+---
+
+# ACL Roles HTTP API
+
+The `/acl/roles` and `/acl/role/` endpoints are used to manage ACL Roles.
+
+## Create Role
+
+This endpoint creates an ACL Role. The request is always forwarded to the
+authoritative region.
+
+| Method | Path        | Produces           |
+| ------ | ----------- | ------------------ |
+| `POST` | `/acl/role` | `application/json` |
+
+The table below shows this endpoint's support for
+[blocking queries](/api-docs#blocking-queries) and
+[required ACLs](/api-docs#acls).
+
+| Blocking Queries | ACL Required |
+| ---------------- | ------------ |
+| `NO`             | `management` |
+
+### Parameters
+
+- `Name` `(string: <required>)` - Specifies the human-readable name of the ACL
+  Role. The name can contain alphanumeric characters, dashes, and underscores.
+  This name must be unique and must not exceed 128 characters.
```

Comment:
Just double-checking, ACLs are not namespaced right? So this needs to be globally unique in the cluster and across regions?

Reply:
No, ACLs are not namespaced meaning names are globally unique due to replication.